modal/sandbox.py

```diff
@@ -495,6 +496,22 @@ async def exec(
         by_line = bufsize == 1
         return _ContainerProcess(resp.exec_id, self._client, stdout=stdout, stderr=stderr, text=text, by_line=by_line)
 
+    async def open(self, path: str, mode: str = "r") -> _FileIO:
+        """Open a file in the Sandbox and return
+        a [`FileIO`](/docs/reference/modal.FileIO#modalfile_io) handle.
```

Comment:
This link should be a full domain so people using the client library directly can hit the link.

Reply:
it should yeah